Sofa/Component/Collision/Response/Contact/src/sofa/component/collision/response/contact/CollisionResponse.cpp

```diff
@@ -62,19 +62,33 @@ sofa::helper::OptionsGroup CollisionResponse::initializeResponseOptions(sofa::co
     }
 
     sofa::helper::OptionsGroup responseOptions(listResponse);
-    if (listResponse.contains("PenalityContactForceField"))
-        responseOptions.setSelectedItem("PenalityContactForceField");
-
     return responseOptions;
 }
 
 void CollisionResponse::init()
 {
+
     Inherit1::init();
+
+    if(!d_response.isSet())
+    {
+        msg_error() << "No response method has been set";
+        return;
+    }
+
     if (d_response.getValue().size() == 0)
     {
+        msg_error() << "Response method is empty and may have been wrongly set. Option list is: " << initializeResponseOptions(getContext());
         d_response.setValue(initializeResponseOptions(getContext()));
     }
+    else
+    {
+        sofa::helper::OptionsGroup responseOptions = initializeResponseOptions(getContext());
+        if(!responseOptions.isInOptionsList(d_response.getValue().getSelectedItem()))
```

Comment:
This is the source of your multiple (wrong) errors message on the scene tests.
`isInOptionsList()` does not return a boolean (as its name would suggest) but the index in the list if found (and -1 is not found). So if it finds the first element in the list, it will return 0 and your test will fail 🫢
```suggestion
        if(responseOptions.isInOptionsList(d_response.getValue().getSelectedItem()) < 0)
```

Obviously there is some fix/renaming to do in OptionsGroup 🥴